README.md

```diff
@@ -43,7 +44,7 @@ Feel free to simply copy and paste the code snippets in the notebooks. If you pr
    WRITER_API_KEY="{Your Writer API key goes here}"
    ```
 
-## Running the cookbooks
+## Run the cookbooks
```

Comment:
FYI we currently use gerunds consistently in the docs for these headers. Not a dealbreaker here but just so you know it was intentional.